validatorclient/src/main/java/tech/pegasys/artemis/validatorclient/ValidatorClient.java

```diff
@@ -13,7 +13,68 @@
 
 package tech.pegasys.artemis.validatorclient;
 
+import static tech.pegasys.artemis.datastructures.util.BeaconStateUtil.get_crosslink_committees_at_slot;
+import static tech.pegasys.artemis.datastructures.util.BeaconStateUtil.get_current_epoch;
+import static tech.pegasys.artemis.datastructures.util.BeaconStateUtil.get_epoch_start_slot;
+import static tech.pegasys.artemis.datastructures.util.BeaconStateUtil.get_previous_epoch;
+
+import com.google.common.primitives.UnsignedLong;
+import java.util.ArrayList;
+import java.util.List;
+import tech.pegasys.artemis.datastructures.state.BeaconState;
+import tech.pegasys.artemis.datastructures.state.CrosslinkCommittee;
+
 public class ValidatorClient {
 
+  private static final int SLOTS_PER_EPOCH = 64; // 2^6
```

Comment:
It might be a good idea to put this constant in constants file and import from constants.

Reply:
The name had been updated from an old constant - updating constant names and removing this line.